model.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Un-intended placement of this model file?

Reply:
good catch. thank you